src/algorithms/pid/IrtCherenkovParticleID.h

```diff
@@ -59,17 +59,17 @@ class IrtCherenkovParticleID : public IrtCherenkovParticleIDAlgorithm,
   void process(const Input&, const Output&) const;
 
 private:
+  // any access (R or W) to m_irt_det_coll, m_irt_det, m_pid_radiators must be locked
+  inline static std::mutex m_irt_det_mutex;
```

Comment:
Do we expect that DRICH and pfRICH geometries could be operated on orthogonally?